lib/facade_converter.ts

```diff
@@ -285,11 +294,14 @@ export class FacadeConverter extends base.TranspilerBase {
   private TS_TO_DART_TYPENAMES: ts.Map<ts.Map<string>> = {
     'lib': this.stdlibTypeReplacements,
     'lib.es6': this.stdlibTypeReplacements,
-    'typings/es6-promise/es6-promise': {'Promise': 'Future'},
     'angular2/typings/es6-promise/es6-promise': {'Promise': 'Future'},
```

Comment:
if we default typingsRoot to angular2/typings then we could remove a couple of these?